setup.cfg

```diff
@@ -3,6 +3,5 @@ current_version = 0.11.0
 commit = True
 tag = True
 
-[bumpversion:file:setup.py]
-search = VERSION = "{current_version}"
-
+[bumpversion:file:d3a_api_client/__init__.py]
+search = __version__ = "{current_version}"
```

Comment:
Nit: it would be better to restore the newline here. On PyCharm 2021 you can find the setting here:
> `Settings` > `Editor` > `General` > `Ensure every saved file ends with a line break`.